tests/test_wave_01.py

```diff
@@ -159,12 +159,18 @@ def test_moves_movie_from_watchlist_to_empty_watched():
     assert len(updated_data["watchlist"]) == 0
     assert len(updated_data["watched"]) == 1
     
-    raise Exception("Test needs to be completed.")
+    if updated_data["watched"][0] != {
+        "title": MOVIE_TITLE_1,
+        "genre": GENRE_1,
+        "rating": RATING_1
+        }:
+        raise Exception("The movie was not moved to the watched.")
```

Comment:
How this be rewritten with using an `assert` statement so that we follow the testing format we use with Arrange, Act, Assert?